packages/components/src/styles/components/accordion.scss

```
@@ -46,68 +110,103 @@ $hds-accordion-item-border-radius: 6px;
 .hds-accordion-item__toggle {
   position: relative;
   display: flex;
-  gap: 12px;
+  gap: $hds-accordion-item-gap;
   align-items: center;
-  padding:
-    $hds-accordion-item-padding
-    $hds-accordion-item-padding
-    $hds-accordion-item-padding
+
+  .hds-accordion-item--size-large & {
+    padding:
+    $hds-accordion-item-padding-large
+    $hds-accordion-item-padding-large
+    $hds-accordion-item-padding-large
+    12px; // by design
+  }
+
+  .hds-accordion-item--size-medium & {
+    padding:
+    $hds-accordion-item-padding-medium
+    $hds-accordion-item-padding-medium
+    $hds-accordion-item-padding-medium
     12px; // by design
+  }
+
+  .hds-accordion-item--size-small & {
+    padding:
+    $hds-accordion-item-padding-small
+    $hds-accordion-item-padding-small
+    $hds-accordion-item-padding-small
+    12px; // by design
+  }
 }
 
 .hds-accordion-item__button {
   padding: 0;
 
   &:hover { cursor: pointer; }
 
-  // entire toggle area is interactive
-  &.hds-accordion-item__button--parent-does-not-contain-interactive {
-    @include hds-focus-ring-with-pseudo-element();
-    position: static;
-    margin: -1px 0;
-    color: var(--token-color-foreground-primary);
-    background: transparent;
-    border: 1px solid transparent;
-
-    // expand button target to cover entire AccordionItem Toggle block (depending on the `@containsInteractive/@parentContainsInteractive` argument)
-    &::after {
-      position: absolute;
-      display: block;
-      border-radius: $hds-accordion-item-border-radius;
-      content: "";
-      inset: 0;
+  .flight-icon-chevron-down {
```

In hindsights, we can use a less specific class name in this case
```suggestion
  .flight-icon {
```